src/presentation-markup.html

```diff
@@ -1063,8 +1063,90 @@ <h5 id="presm_presatt_core">MathML Core Attributes</h5>
        background color, if any, used in the current rendering context to show through.
       </td>
      </tr>
+
```

Comment:
If you add to this table rather than make a new one, then need to restructure the leading  paragraph

> These attributes include the following two attributes that are primarily intended for visual media. ....

as it now has more than two entries and not all for same usage.